src/Project/BuilderCfgFile.ts

```diff
@@ -72,6 +72,10 @@ export class BuilderCfgFile extends EventEmitter implements helpers.FileSelector
     this.on(K_BEGIN_IMPORT, this.onBeginImport);
   }
 
+  public importCfg(parsedCfg: any) {
```

Comment:
It would be better to split `Config` and `Project` codes.

Reply:
I made this method because I wanted to call `onBeginImport` from `RunConfig.ts`. But the method was private, so I couldn't.
So I made `importCfg` that I couldn't find other way

Reply:
I mean, split PRs...
Anyway this PR is a good example for experiment but as Project.Builder is not prepared to accept Objects from other modules yet. And I don't have time for this as of now.